p3/utils.py

```diff
@@ -332,7 +332,7 @@ def group_all_talks_by_admin_type(conference, talk_status='accepted'):
                        'helpdesk':    ['h_180'],
                       }
 
-        for grp_name, grp_types in type_groups.items():
+        for grp_name, grp_types in list(type_groups.items()):
```

Comment:
Unnecessary `list`.

Reply:
For future reference: find . -type f -name \*.py -exec gawk -i inplace '{print gensub(/for (.+) in list\((.+)\):/, "for \\1 in \\2:", "g");}' {} \;